doc/spire_agent.md

```diff
@@ -255,14 +254,16 @@ plugins {
 
 ## Envoy SDS Support
 
-SPIRE agent has **beta** support for the [Envoy](https://envoyproxy.io) [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret) (SDS).
-When enabled, SDS is served over the same Unix domain socket as the Workload API. Envoy processes connecting to SDS are attested as workloads.
+SPIRE agent has support for the [Envoy](https://envoyproxy.io) [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret) (SDS).
+SDS is served over the same Unix domain socket as the Workload API. Envoy processes connecting to SDS are attested as workloads.
 
 [`auth.TlsCertificate`](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/auth/cert.proto#envoy-api-msg-auth-tlscertificate)
-resources containing X.509-SVIDs can be fetched using the SPIFFE ID of the workload as the resource name (e.g. `spiffe://example.org/database`).
+resources containing X.509-SVIDs can be fetched using the SPIFFE ID of the workload as the resource name (e.g. `spiffe://example.org/database`). Alternatively, if
+requesting the default `auth.TlsCertificate`, the default name "default" may be used. The default name is configurable.
```

Comment:
We may be more explicit and point to the configuration setting?
Something like "The default name is configurable through the `default_svid_name` setting inside the `sds` configuration section".

Reply:
Sure, I can do that.